Cargo.toml

```
@@ -12,10 +12,13 @@ categories = []
 keywords = []
 
 [features]
-default = ["track-caller", "capture-spantrace"]
+default = ["track-caller", "capture-spantrace", "auto-install", "track-caller"]
```

```suggestion
default = ["auto-install", "capture-spantrace", "track-caller"]
```

`"track-caller"` was in there twice. It's easier to spot in alphabetic order.

---

https://github.com/eyre-rs/eyre/pull/138 introduces `"anyhow"` as new `eyre` default feature flag.

https://github.com/eyre-rs/eyre/issues/136 intends to make the new `"anyhow"` feature non-`default` in a breaking release.